src/com/shmuelzon/HomeAssistantFloorPlan/Controller.java

```diff
@@ -890,11 +890,11 @@ private void generateSensorEntities(Map<String, Entity> homeAssistantEntities) {
     }
 
     private boolean doStateIconsIntersect(Entity first, Entity second) {
-        final double STATE_ICON_RAIDUS_INCLUDING_MARGIN = 25.0;
+        final double STATE_ICON_RADIUS_INCLUDING_MARGIN = 20.0;
```

Comment:
Yes, the icons are 40px but, as the variable states, I'm including a margin of 5 (actually 10px) since I didn't want them to be too close, or even touch.
Please remove this change.

Reply:
My bad. I assumed it regarded the element in the browser, which has a margin of `0`.
Now I understand you mean the space between the icons.

I'll change it back, but maybe it can be implemented as a user-setting with a separate PR?
E.g. I like the icons being 1 or 2px apart.

Reply:
I think that'll be a bit more fine-tuning than actually needed. At the end of the day, I don't want to offer too many options since it my become overwhelming. Even hiding it in an advanced options window is something I prefer to postpone until I have no other option.

I don't remember how I got to the 5px margin in the first place, probably just tried a few values and saw what looks good to me, If this ever becomes and issue to more people, we can consider changing it.